### PR TITLE
Fix infinite looping when session.close() called (#161)

### PR DESCRIPTION
**Pull request by Algy**

### CODE REVIEW

File: sockjs/transports/rawwebsocket.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def server(self, ws, session):
 
     @asyncio.coroutine
     def client(self, ws, session):
+        closing = getattr(web.MsgType, 'closing', None)
```
Comment (fafhrd91) [inline sockjs/transports/rawwebsocket.py]: just use `MsgType.closing` and bump aiohttp requirements in setup.py

**Comment by asvetlov** [comment]
The patch is great but please add a test

**Comment by Algy** [comment]
Sure! I'll come up with tests + revise according to @fafhrd91's review this weekend.

**Comment by jettify** [comment]
there is other PR that contents fixes from this one https://github.com/aio-libs/sockjs/pull/165/files

**Pull request merged**